libdlib/CMakeLists.txt

```diff
@@ -17,7 +17,7 @@ ExternalProject_Add(EP_${PROJECT_NAME}
     BINARY_DIR ${PROJECT_BINARY_DIR}/${PROJECT_NAME}-build
     
     CONFIGURE_COMMAND ${CMAKE_COMMAND} ${PROJECT_BINARY_DIR}/${PROJECT_NAME}-src/dlib
-    BUILD_COMMAND ${CMAKE_COMMAND} --build . --config Release
+    BUILD_COMMAND ${CMAKE_COMMAND} --build . --config Release -- -Wno-deprecated-declarations -Wno-terminate
```

Comment:
if you want to silence `libphidgets`, why are you specifying this for `libdlib`?

Reply:
I want to silence warnings from both packages...it works for libdlib...with this pr....
now I wanted to add a commit to this pr which silences the warnings issued by libphidgets...

in the end, I want them both silenced :wink:

Reply:
Ah, "also" ;)

Reply:
What happens, if you specify this for libphidgets as well?

Reply:
well, `libphidgets` uses [`make` as `BUILD_COMMAND`](https://github.com/ipa320/cob_extern/blob/indigo_dev/libphidgets/CMakeLists.txt#L17) while `libdlib` uses [`cmake` as `BUILD_COMMAND`](https://github.com/ipa320/cob_extern/blob/indigo_dev/libdlib/CMakeLists.txt#L20)

Reply:
I see. Normally, you would pass it in the configuration step anyway. E.g. by specifying `CXXFLAGS` and/or `CFLAGS`.